ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java

```diff
@@ -74,27 +76,40 @@ public boolean isTopicExists(String topic) {
 
   public Set<String> listTopicNames() {
     try {
-      AdminClient adminClient = AdminClient.create(adminClientConfig);
-      Set<String> topicNames =  adminClient.listTopics().names().get();
-      adminClient.close();
-      return topicNames;
+      return adminClient.listTopics().names().get();
     } catch (InterruptedException | ExecutionException e) {
       throw new KafkaResponseGetFailedException("Failed to retrieve kafka topic names", e);
     }
   }
 
   public Map<String, TopicDescription> describeTopics(Collection<String> topicNames) {
     try {
-      AdminClient adminClient = AdminClient.create(adminClientConfig);
-      Map<String, TopicDescription> topicInfos =  adminClient.describeTopics(topicNames).all()
+      return adminClient.describeTopics(topicNames).all()
           .get();
-      adminClient.close();
-      return topicInfos;
     } catch (InterruptedException | ExecutionException e) {
       throw new KafkaResponseGetFailedException("Failed to describe kafka topics", e);
     }
   }
 
+  public void deleteTopics(List<String> topicsToDelete) {
```

Comment:
tests for this method?

Reply:
Will add test when I finish the clean up task which this method is for.

Reply:
as above